examples/cloudevents/main.py

```diff
@@ -12,6 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# This sample creates a function that accepts a Cloud Event per
+# https://github.com/cloudevents/sdk-python
+import sys
 
-def hello(data, context):
-    pass
+
+def hello(cloudevent):
+    print("Received event with ID: %s" % cloudevent.EventID(), file=sys.stdout, flush=True)
+    return 200
```

Comment:
```suggestion
```
Since we're not responding with the return value of the function, the implicit return is fine here.